api/tests/unit/application/users/index_test.js

```diff
@@ -255,4 +255,26 @@ describe('Unit | Router | user-router', () => {
       });
     });
   });
+
+  describe('GET /api/users/{id}/organizations', function() {
+    beforeEach(() => {
+      sinon.stub(userController, 'getOrganizations').returns('ok');
+      sinon.stub(securityController, 'checkRequestedUserIsAuthenticatedUser').callsFake((request, h) => h.response(true));
+      startServer();
+    });
+
+    it('should exist', () => {
```

Comment:
Est-ce que ces tests sont utiles, sachant que les tests d'acceptances valident déjà l'existence de la route ?

Reply:
Honnêtement je reproduis ici toujours ces tests unitaires par imitation avec ce qui est déjà fait, mais je dois dire que j'ai l'impression qu'on teste plusieurs fois plein de choses.... J'aimerais porter cette discussion sur un sujet plus global sur les tests.

Reply:
Oui, pour moi le 'should exist' est du bruit à éviter.